### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by anastasiiashchoholieva**

### CODE REVIEW

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]
all tests should pass, there is no needed files in ur PR

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        self.number = number
+
+    def movie_session(
+        self,
+        movie_name: str,
+        customers: list,
+        cleaning_staff: str
+    ) -> str:
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        [customer.watch_movie(movie_name) for customer in customers]
```
Comment (ArsenPidhoretskyi) [inline app/cinema/hall.py:12]: In this way better will be
`list(map(lambda customer: customer.watch_movie(movie_name), customers)) `

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
-    pass
+def cinema_visit(
+    customers: list,
+    hall_number: int,
+    cleaner: str,
+    movie: str
+) -> str:
+    customer_list = []
+    for customer in customers:
+        customer_list.append(Customer(customer["name"], customer["food"]))
```
Comment (ArsenPidhoretskyi) [inline app/main.py:15]: Use list comprehensions, also customer_list is not good name
Better customers, but this name already used, than customers_objects

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+    for people in customer_list:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:16]: people -> customer

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+        CinemaBar.sell_product(people, people.food)
+
+    cinema_order = CinemaHall(hall_number)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:19]: cinema_order -> cinema_hall or hall

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 5]
```diff
@@ -1,6 +1,24 @@
...
+    movie: str
+) -> str:
+    customers_objects = [
+        Customer(customer["name"], customer["food"])
+        for customer in customers
+    ]
+    [
+        CinemaBar.sell_product(customer, customer.food)
+        for customer in customers_objects
+    ]
```
Comment (ArsenPidhoretskyi) [inline app/main.py:20]: In the future use list(map()) as I say before

**Review by ArsenPidhoretskyi — APPROVED** [review]
Good!

**Pull request closed without merging**